CHANGELOG.md

```diff
@@ -67,6 +69,7 @@
 * [BUGFIX] Ingester: Fix race condition in per-tenant TSDB creation. #9708
 * [BUGFIX] Ingester: Fix race condition in exemplar adding. #9765
 * [BUGFIX] Ingester: Fix race condition in native histogram appending. #9765
+* [BUGFIX] PromQL: Fix round function not honoring delayed name removal flag. #9879
```

Comment:
There's no delayed name removal flag in Mimir, so this doesn't apply here.

Reply:
Although I see that this isn't accurate, actually the behaviour changes now, and `round()` now drops the metric name? This looks like a breaking change.

Reply:
`round()` always dropped the metric name; it was mistakenly changed to not do that in 2.55.0 and fixed in 2.55.1.

Reply:
Hum, what did we ship with 2.14?

Reply:
Good point, let me compare to the previous release.

Reply:
I fixed the changelog entry so it's accurate. I compared to v2.14.1 and can see that it's a fix versus that version.

Reply:
@colega WDYT? Should it still be a BUGFIX, or a CHANGE instead?